.github/workflows/pr.yaml

```diff
@@ -1,40 +1,25 @@
-name: Pull Request workflow
+name: Pull Request Workflow
+
 on:
   pull_request:
-    branches:
-      - master
+    branches: [ main ]
+
 jobs:
-  test:
-    name: Tests
+  validate-configs:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        experiment:
-          # - locust-metrics
-          #- voting-webapp
-          #- jvm
-          #- parallel-trials
-          #- postgres
-          #- elasticsearch
+
     steps:
-      - name: Check out code
-        uses: actions/checkout@v2
-      - name: KinD (Kubernetes in Docker) Initialization
-        uses: helm/kind-action@v1.0.0-rc.1
-        with:
-          version: v0.8.1
-      - name: Download and Setup redskyctl
-        run: |
-          .ci/setup_redskyctl.sh
-      - name: Install kustomize
-        run: |
-          .ci/install_kustomize.sh
-      - name: Run sample trial for ${{ matrix.experiment }}
-        run: |
-          .ci/${{ matrix.experiment }}/ci.sh
-      - name: The job has failed
-        if: ${{ failure() }}
-        run: |
-          kubectl get trial,experiment,svc,pod -o wide
-          kubectl get pods -o wide
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install kustomize
+      run: |
+        wget -q "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.2.0/kustomize_v4.2.0_linux_amd64.tar.gz" -O - | tar xz
+        sudo mv kustomize /usr/local/bin/kustomize
 
+    - name: Build all included kustomizations
+      run: |
+        set -eo pipefail
+        find . -type f -maxdepth 6 -name kustomization.yaml \
+        | sed 's/\/kustomization.yaml//g' \
```

Comment:
super nit and won't be hung up, but this might be a little more appropriate `| xargs -n1 dirname \`